content/en/docs/ops/configuration/security/migrate-alpha-policy/index.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Migrate the deprecated alpha security policy
+description: A tutorial to help customers migrate the deprecated alpha security policy to the beta version.
+keywords: [security,policy,migrate,alpha,beta,deprecate,peer,jwt,authorization]
+weight: 20
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+Istio beta security policy (`PeerAuthentication`, `RequestAuthentication` and `AuthorizationPolicy`) has been released
+since release 1.5 and the alpha security policy (`MeshPolicy`, `Policy`, `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`)
+has been deprecated and no longer supported starting with release 1.6.
+
+It is required to migrate the alpha security policy to the beta version before upgrading to Istio 1.6 and later.
+This tutorial helps customers who are still using the alpha security policy in a `pre-1.6` version to migrate to
+the beta security policy and unblock the upgrade to Istio 1.6 and later.
+
+## Overview
+
+The control plane should be upgraded first to bring in support of the `v1beta1` security policy. It is recommended to first
+upgrade to Istio 1.5 as a transitive version because it supports both `v1alpha1` and `v1beta1` security policies. You complete
+the security policy migration in Istio 1.5, remove the `v1alpha1` security policy and then continue the upgrade to later Istio versions.
+
+Istio 1.5 is the only version that supports both `v1alpha1` and `v1beta1` security policy. For a given workload, the
+`v1beta1` version will take precedence over the `v1alpha1` version.
+
+Alternatively, if you want to do a skip-upgrade directly from Istio 1.4 to Istio 1.6 or later, you should use the
+[canary upgrade](/docs/setup/upgrade/canary/) to install the new Istio version as a separate control plane, gradually
+migrate your workloads to the new control plane completing the security policy migration at the same time.
+
+Note that skip-upgrade is not supported by Istio and there might be other issues in this process. Istio 1.6 does not support
```

Comment:
Is it worth putting a warning callout around this paragraph? When I get to the `In either case,` paragraph I look back and see 3 paragraphs so expects 3 different cases, so would like to combine the skip-level paragraphs into one. But then the warning is sort of lost. Having it as a callout doesn't leave an extra paragraph but still causes the reader to not miss the waring.

Reply:
Yeah, that makes sense. I changed to use warning around this and also updated the previous a little bit, it now should be more clear about the 2 cases.